common/html-utils/src/main/resources/xml/xslt/html-id-fixer.xsl

```diff
@@ -49,9 +46,8 @@
         </xsl:copy>
     </xsl:template>
 
-
     <xsl:template match="*[@epub:type='pagebreak']">
-        <!--TODO FIXME: epub:type can have several values-->
+        <!-- FIXME: epub:type can have several values -->
```

Comment:
> maybe it could be fixed altogether

Yeah OK. Indeed not very hard to fix. But I didn't want to get too side-tracked. The amount of side-tracking was already enormous :-), but most other changes to the behavior that I did were needed for the dtbook-to-epub3 rewrite.

> ultimately this code would need to be adapted to also support DPUB ARIA roles

Can you create a new issue for this? Or do we have one for it?

Reply:
> Can you create a new issue for this?

Done in daisy/pipeline-tasks#140